src/XrdOfs/XrdOfs.cc

```diff
@@ -2252,10 +2284,29 @@ int XrdOfs::rename(const char             *old_name,  // In
        evsObject->Notify(XrdOfsEvs::Mv, evInfo);
       }
 
+// If we cannot overwrite, we must open-exclusive first.  This will test whether
```

Comment:
Could you correct the comment? We no longer do an excl open here.

Reply:
Done.